.github/workflows/ci-tests.yml

```
@@ -47,37 +47,33 @@ jobs:
           go-version-file: go.mod
           cache-dependency-path: go.sum
 
-      - name: Cache
-        uses: actions/cache@v4
-        with:
-          path: |
-            ~/.cache/golangci-lint
-          key: 'golangci-lint-${{ runner.os }}-${{ hashFiles(''**/go.sum'') }}'
-          restore-keys: |
-            golangci-lint-${{ runner.os }}-
-
       - name: Setup CI Tooling
         uses: shrink/actions-docker-extract@v3
         with:
           image: tykio/ci-tools:latest
-          path: /usr/local/bin/golangci-lint
+          path: /usr/local/bin/task
```

this needs other deps
```suggestion
          path: /usr/local/bin/.
```